tasks/gen/action.cr

```diff
@@ -63,4 +35,24 @@ class Gen::Action < LuckyCli::Task
     @error = "That's not a valid Action.  Example: lucky gen.action Users::Index"
```

Comment:
I didn't notice this before, but there are 2 spaces between Action and Example

Reply:
Good catch.